tests/functional/test_wps_package.py

```diff
@@ -134,6 +141,11 @@ def test_literal_io_from_package(self):
         assert "maxOccurs" not in desc["process"]["outputs"][0]
         assert "format" not in desc["process"]["outputs"][0]
         assert len(set(desc["process"].keys()) - KNOWN_PROCESS_DESCRIPTION_FIELDS) == 0
+        # make sure that deserialization of literal fields did not produce over-verbose metadata
+        for p_input in desc["process"]["inputs"]:
+            assert len(set(p_input) - KNOWN_PROCESS_DESCRIPTION_INPUT_DATA_FIELDS) == 0
```

Comment:
Using directly `assert not set1 - set2` might be a good way to verify too

Reply:
not critical